content/zh/docs/tutorials/kubernetes-basics/cluster-intro.html

```diff
@@ -29,7 +29,7 @@ <h3>目标</h3>
 
             <div class="col-md-8">
                 <!-- <h3>Kubernetes Clusters</h3> -->
-                <h3> Kubernetes 集群</h3>
+                <h3>Kubernetes集群</h3>
```

Comment:
We have code style as space between english, number and such things :D
```suggestion
                <h3>Kubernetes 集群</h3>
```